src/resolve.rs

```diff
@@ -99,25 +99,9 @@ impl ResolveValue for ast::Expression {
                 ref variants,
             } => select_default(variants).and_then(|variant| variant.value.to_value(env)),
             ast::Expression::SelectExpression {
-                expression: Some(box ast::Expression::MessageReference { ref id }),
+                expression: Some(box ast::Expression::MessageReference { .. }),
```

Comment:
Is there a reason to still special-case `MessageReference` used as a selector? There's another `ast::Expression::SelectExpression` branch in this `match` right below and I'd think it's enough?